src/__tests__/Path.test.tsx

```diff
@@ -47,6 +47,21 @@ describe('Path', () => {
     });
   });
 
+  it('Render all segments by default', () => {
+    const { container } = render({});
+    const paths = container.querySelectorAll('path');
+    expect(paths).toHaveLength(dataMock.length);
+  });
+
+  it('Do not render a segment', () => {
+    const firstDataEntry = dataMock[0];
+    const { container } = render({
+      filterSegments: dataEntry => dataEntry.value === firstDataEntry.value,
+    });
+    const paths = container.querySelectorAll('path');
+    expect(paths).toHaveLength(dataMock.length - 1);
+  });
+
```

Comment:
Since the feature was implemented in `Chart/renderSegments.tsx` I guess `src/__tests__/Chart.test.tsx` is the right place for this test.

I'm pretty sure the first test case is already covered by the existing tests.

I'd suggest to follow the same structure as the existing tests. Something like:

```
describe('"filterSegments"', () => {
  it('test description', () => {
    // Implementation
  });
});
```